Putting in /money deposit, /money pay, /money send and so on throws an error in the console instead of "please specify the amount"

**Describe the bug**
Trying out the plugin and I love it so far. Testing out the commands without values throws an error every time. In the game as well as the console.

**To Reproduce**
Steps to reproduce the behavior:
1. Put in any command that would expect a value without specifying one.
2. See error

**Expected behavior**
There should be no error. Just a message asking to specify a value.

**Screenshots**
Has been added.

**Desktop (please complete the following information):**
 - Server Type: [PaperMC]
 - Server Version: [git-Paper-61 (MC: 1.19) (Implementing API version 1.19-R0.1-SNAPSHOT) (Git: cd215af)]]
 - Gringotts Version: [v2.12.4-SNAPSHOT]

**Additional context**
![2023-04-13_12 49 15](https://user-images.githubusercontent.com/109665179/231737685-81895783-b0c1-4f46-83d8-f451afc5d67e.png)

[12:39:19] [Server thread/INFO]: Haui111 issued server command: /money deposit
[12:39:19] [Server thread/ERROR]: null
org.bukkit.command.CommandException: Unhandled exception executing command 'money' in plugin Gringotts v2.12.4-SNAPSHOT
	at org.bukkit.command.PluginCommand.execute(PluginCommand.java:47) ~[paper-api-1.19-R0.1-SNAPSHOT.jar:?]
	at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:155) ~[paper-api-1.19-R0.1-SNAPSHOT.jar:?]
	at org.bukkit.craftbukkit.v1_19_R1.CraftServer.dispatchCommand(CraftServer.java:909) ~[paper-1.19.jar:git-Paper-61]
	at net.minecraft.server.network.ServerGamePacketListenerImpl.handleCommand(ServerGamePacketListenerImpl.java:2376) ~[?:?]
	at net.minecraft.server.network.ServerGamePacketListenerImpl.lambda$handleChatCommand$18(ServerGamePacketListenerImpl.java:2147) ~[?:?]
	at net.minecraft.server.TickTask.run(TickTask.java:18) ~[paper-1.19.jar:git-Paper-61]
	at net.minecraft.util.thread.BlockableEventLoop.doRunTask(BlockableEventLoop.java:153) ~[?:?]
	at net.minecraft.util.thread.ReentrantBlockableEventLoop.doRunTask(ReentrantBlockableEventLoop.java:24) ~[?:?]
	at net.minecraft.server.MinecraftServer.doRunTask(MinecraftServer.java:1358) ~[paper-1.19.jar:git-Paper-61]
	at net.minecraft.server.MinecraftServer.d(MinecraftServer.java:183) ~[paper-1.19.jar:git-Paper-61]
	at net.minecraft.util.thread.BlockableEventLoop.pollTask(BlockableEventLoop.java:126) ~[?:?]
	at net.minecraft.server.MinecraftServer.pollTaskInternal(MinecraftServer.java:1335) ~[paper-1.19.jar:git-Paper-61]
	at net.minecraft.server.MinecraftServer.pollTask(MinecraftServer.java:1328) ~[paper-1.19.jar:git-Paper-61]
	at net.minecraft.util.thread.BlockableEventLoop.managedBlock(BlockableEventLoop.java:136) ~[?:?]
	at net.minecraft.server.MinecraftServer.waitUntilNextTick(MinecraftServer.java:1306) ~[paper-1.19.jar:git-Paper-61]
	at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1191) ~[paper-1.19.jar:git-Paper-61]
	at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:302) ~[paper-1.19.jar:git-Paper-61]
	at java.lang.Thread.run(Thread.java:833) ~[?:?]
Caused by: java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at org.gestern.gringotts.commands.MoneyExecutor.onCommand(MoneyExecutor.java:66) ~[gringotts-2.12.4-SNAPSHOT.jar:?]
	at org.bukkit.command.PluginCommand.execute(PluginCommand.java:45) ~[paper-api-1.19-R0.1-SNAPSHOT.jar:?]
	... 17 more

--- a/src/main/java/org/gestern/gringotts/Language.java
+++ b/src/main/java/org/gestern/gringotts/Language.java
@@ -26,6 +26,7 @@ public enum Language {
     public String invalid_account;
     public String reload;
     public String added_denomination;
+    public String invalid_number;
 
     // Adding a denomination
     public String hold_item;
@@ -114,6 +115,9 @@ public enum Language {
         LANG.denomination_value = translator.apply(
                 "errors.denominationValue",
                 "Invalid argument '%value' is not a valid number for denomination value");
+        LANG.invalid_number = translator.apply(
+                "errors.invalidNumber",
+                "Invalid argument: '%value' is not a number!");
 
         //pay command
         LANG.pay_success_sender = translator.apply(

--- a/src/main/java/org/gestern/gringotts/commands/MoneyExecutor.java
+++ b/src/main/java/org/gestern/gringotts/commands/MoneyExecutor.java
@@ -47,6 +47,8 @@ public class MoneyExecutor extends GringottsAbstractExecutor {
             return true;
         }
 
+        if (args.length < 2) return false;
+
         String command = args[0];
 
         switch (command.toLowerCase()) {
@@ -58,7 +60,8 @@ public class MoneyExecutor extends GringottsAbstractExecutor {
 
                     return true;
                 } catch (NumberFormatException ignored) {
-                    return false;
+                    player.sendMessage(Language.LANG.invalid_number.replace("%value", args[1]));
+                    return true;
                 }
             }
             case "deposit": {
@@ -69,27 +72,32 @@ public class MoneyExecutor extends GringottsAbstractExecutor {
 
                     return true;
                 } catch (NumberFormatException ignored) {
-                    return false;
+                    player.sendMessage(Language.LANG.invalid_number.replace("%value", args[1]));
+                    return true;
                 }
             }
             case "send": {
+                if (args.length < 3) return false;
                 try {
                     double value = Double.parseDouble(args[2]);
 
                     // money send <player> <amount>
                     return pay(player, value, args[1]);
                 } catch (NumberFormatException ignored) {
-                    return false;
+                    player.sendMessage(Language.LANG.invalid_number.replace("%value", args[2]));
+                    return true;
                 }
             }
             case "pay": {
+                if (args.length < 3) return false;
                 try {
                     double value = Double.parseDouble(args[1]);
 
                     // money pay <amount> <player>
                     return pay(player, value, args[2]);
                 } catch (NumberFormatException ignored) {
-                    return false;
+                    player.sendMessage(Language.LANG.invalid_number.replace("%value", args[1]));
+                    return true;
                 }
             }
         }

--- a/src/main/resources/i18n/messages_fr.yml
+++ b/src/main/resources/i18n/messages_fr.yml
@@ -6,6 +6,15 @@ inv_balance: "Solde de l'inventaire: %balance"
 end_balance: "Solde de l'enderchest: %balance"
 invalidaccount: "Compte invalide : %player"
 reload: "Gringotts: Configuration rechargée !"
+added_denomination: "Coupure ajoutée : %name"
+
+errors:
+    denominationValue: "valeur de la coupure"
+    denominationName: "nom de la coupure"
+    holdItem: "Veuillez tenir un objet"
+    missingValue: "Argument manquant: %name"
+    invalidValue: "Argument invalide '%value' n'est pas un nombre valide pour %name"
+    invalidNumber: "Argument invalide : '%value' n'est pas un nombre !"
 
 pay:
     success:

--- a/src/main/resources/messages.yml
+++ b/src/main/resources/messages.yml
@@ -25,6 +25,7 @@ errors:
     holdItem: "Please hold an item"
     missingValue: "Missing argument: %name"
     invalidValue: "Invalid argument '%value' is not a valid number for %name"
+    invalidNumber: "Invalid argument: '%value' is not a number!"
 
 pay:
     success:

